30_regexp_lab/relab.txt

```diff
@@ -1,17 +1,19 @@
 Use grep to print only the lines from "input" for which:
 
 1. There is a binary number in the line
+	cat input |grep -a [0-9]
```

Comment:
A binary number is one that has 0s and 1s only. For example the line `The number 01010100 is binary` has a binary number while the line `The number 90 is not` does not have one.

A regular expression would need to check if a single word in the line is completely made up from 0s and 1s only. The grep command for that is: `grep -w [01]+`
Its meaning: Search for a sequence (one or more) of the characters 0 and 1, that appear as a word in itself, due to the -w switch.